Dockerfile-Octave

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04 as builder
```

Comment:
We should probably store these somewhere more organized since I expect we will end up with a few of them them but I am not sure what is best yet so this is fine for now